src/gurt/tests/SConscript

```diff
@@ -14,27 +14,11 @@ def scons():
     if GetOption('help'):
         return
 
-    # pylint: disable=too-many-locals
-    Import('env', 'prereqs', 'gurt_targets')
-    # pylint: enable=too-many-locals
-
-    # Use full path to wrap_cmocka.h for configure test.  Since standalone
-    # cmocka header can't be included without including other headers,
-    # this test can't be generalized
-    wrap_cmocka = os.path.join(Dir('.').srcnode().abspath, 'wrap_cmocka.h')
-    prereqs.define('cmockawrap', headers=[wrap_cmocka], libs=['cmocka'],
-                   package='libcmocka-devel')
-    if not prereqs.check_component('cmocka'):
-        print("\n***************************************************")
-        print("libcmocka-devel package needed to enable unit tests")
-        print("***************************************************\n")
-        # Just returning is the desired behavior here.  The utest target
-        # has yet to be defined so the build will just fail.
-        return
+    Import('env', 'gurt_targets')
 
     test_env = env.Clone()
-    prereqs.require(test_env, "mercury", "uuid", "cmocka")
-    test_env.AppendUnique(LIBS=['pthread', 'm'])
+    test_env.require('mercury', 'uuid')
```

Comment:
why is there no require for cmocka here?

Reply:
There is no provision for building cmocka from source so no munging of include or lib dirs are needed.